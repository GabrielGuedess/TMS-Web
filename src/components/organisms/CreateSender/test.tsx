import { render } from '@testing-library/react';

import { CreateSender } from '.';

const push = jest.fn();

jest.mock('next/navigation', () => ({
  useRouter: () => ({
    push,
  }),
}));

jest.mock('actions/Sender/create', () => ({
  createSender: jest.fn(),
}));

describe('<CreateSender />', () => {
  it('should render the create collaborator correctly', () => {
    const { container } = render(<CreateSender />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
