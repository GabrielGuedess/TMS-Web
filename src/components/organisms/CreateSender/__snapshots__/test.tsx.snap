// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateCollaborator /> should render the create collaborator correctly 1`] = `
<div
  class="flex flex-col gap-6 md:grid md:grid-cols-12"
>
  <div
    class="col-span-4 flex flex-col items-center justify-center gap-6 rounded-2xl border border-zumthor-100 bg-white-lilac-50 px-6 pb-10 pt-20 shadow-default transition-all dark:border-shark-950 dark:bg-smoke-950"
  >
    <div
      class="relative flex size-36 items-center justify-center rounded-full border border-dashed border-dark-200 dark:border-shark-700"
    >
      <svg
        aria-label="Default Photo"
        class="text-comet-500 transition-all dark:text-dark-500"
        fill="none"
        height="56"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="56"
      >
        <path
          d="M20 21v-2a4 4 0 0 0-4-4H8a4 4 0 0 0-4 4v2"
        />
        <circle
          cx="12"
          cy="7"
          r="4"
        />
      </svg>
      <input
        accept="image/*"
        aria-hidden="true"
        aria-label="File"
        class="hidden"
        type="file"
      />
      <button
        aria-label="Change Photo"
        class="group absolute inset-2 z-20 flex cursor-pointer items-center justify-center rounded-full text-white-lilac-50 transition-all hover:bg-black/70 focus:bg-black/70"
        type="button"
      >
        <svg
          class="opacity-0 transition-all group-hover:opacity-100 group-focus:opacity-100"
          fill="none"
          height="36"
          viewBox="0 0 24 24"
          width="36"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6.75992 22H17.2399C19.9999 22 21.0999 20.31 21.2299 18.25L21.7499 9.99C21.8899 7.83 20.1699 6 17.9999 6C17.3899 6 16.8299 5.65 16.5499 5.11L15.8299 3.66C15.3699 2.75 14.1699 2 13.1499 2H10.8599C9.82992 2 8.62992 2.75 8.16992 3.66L7.44992 5.11C7.16992 5.65 6.60992 6 5.99992 6C3.82992 6 2.10992 7.83 2.24992 9.99L2.76992 18.25C2.88992 20.31 3.99992 22 6.75992 22Z"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M10.5 8H13.5"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
          <path
            d="M12 18C13.79 18 15.25 16.54 15.25 14.75C15.25 12.96 13.79 11.5 12 11.5C10.21 11.5 8.75 12.96 8.75 14.75C8.75 16.54 10.21 18 12 18Z"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="1.5"
          />
        </svg>
      </button>
    </div>
    <span
      class="text-center text-xs text-comet-500 transition-all dark:text-dark-300"
    >
      Permitido *.jpeg, *.jpg, *.png, *.gif 
      <br />
      tamanho m√°ximo de 5 Mb
    </span>
    <div
      class="flex gap-4"
    >
      <label
        class="relative flex cursor-pointer items-center gap-2"
        for="active"
      >
        <span
          class="text-sm text-comet-500 dark:text-gray-300"
        >
          Ativo
        </span>
        <button
          aria-checked="true"
          class="peer relative h-5 w-9 rounded-full bg-porcelain-100 transition-all peer-checked:after:translate-x-full peer-focus:outline-none data-[state='checked']:bg-primary-400 dark:border-gray-600 dark:bg-shark-950 data-[state='checked']:dark:bg-primary-400"
          data-state="checked"
          id="active"
          role="switch"
          type="button"
          value="on"
        >
          <span
            class="mx-[2px] flex size-4 rounded-full bg-white-lilac-50 transition-all data-[state='checked']:translate-x-full"
            data-state="checked"
          />
        </button>
      </label>
      <label
        class="relative flex cursor-pointer items-center gap-2"
        for="validated"
      >
        <span
          class="text-sm text-comet-500 dark:text-gray-300"
        >
          Validado
        </span>
        <button
          aria-checked="false"
          class="peer relative h-5 w-9 rounded-full bg-porcelain-100 transition-all peer-checked:after:translate-x-full peer-focus:outline-none data-[state='checked']:bg-primary-400 dark:border-gray-600 dark:bg-shark-950 data-[state='checked']:dark:bg-primary-400"
          data-state="unchecked"
          id="validated"
          role="switch"
          type="button"
          value="on"
        >
          <span
            class="mx-[2px] flex size-4 rounded-full bg-white-lilac-50 transition-all data-[state='checked']:translate-x-full"
            data-state="unchecked"
          />
        </button>
      </label>
    </div>
  </div>
  <form
    aria-label="Form"
    class="col-span-8 flex flex-col justify-between gap-6 rounded-2xl border border-zumthor-100 bg-white-lilac-50 p-6 shadow-default transition-all dark:border-shark-950 dark:bg-smoke-950"
  >
    <div
      class="grid grid-cols-auto-fill gap-x-4 gap-y-10"
    >
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Name"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r0:"
            placeholder="Name"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r0:"
          >
            Nome Completo
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="CPF"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r1:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r1:"
          >
            CPF
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Password"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 py-3 pb-3 pl-3 pr-8 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r2:"
            required=""
            type="password"
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r2:"
          >
            Senha
          </label>
          <button
            aria-label="Visible Password"
            class="absolute right-2 top-1/2 -translate-y-1/2 p-1 text-comet-500 outline-primary-400 transition-all hover:text-primary-400 focus:text-primary-400 dark:text-dark-300 hover:dark:text-primary-400 focus:dark:text-primary-400"
            type="button"
          >
            <svg
              fill="none"
              height="18"
              viewBox="0 0 24 24"
              width="18"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M14.53 9.46998L9.47004 14.53C8.82004 13.88 8.42004 12.99 8.42004 12C8.42004 10.02 10.02 8.41998 12 8.41998C12.99 8.41998 13.88 8.81998 14.53 9.46998Z"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
              <path
                d="M17.82 5.76998C16.07 4.44998 14.07 3.72998 12 3.72998C8.46997 3.72998 5.17997 5.80998 2.88997 9.40998C1.98997 10.82 1.98997 13.19 2.88997 14.6C3.67997 15.84 4.59997 16.91 5.59997 17.77"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
              <path
                d="M8.42004 19.53C9.56004 20.01 10.77 20.27 12 20.27C15.53 20.27 18.82 18.19 21.11 14.59C22.01 13.18 22.01 10.81 21.11 9.39999C20.78 8.87999 20.42 8.38999 20.05 7.92999"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
              <path
                d="M15.5099 12.7C15.2499 14.11 14.0999 15.26 12.6899 15.52"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
              <path
                d="M9.47 14.53L2 22"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
              <path
                d="M22 2L14.53 9.47"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="1.5"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="RG"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r3:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r3:"
          >
            RG
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="CNH"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r4:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r4:"
          >
            CNH
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Email"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r5:"
            required=""
            type="email"
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r5:"
          >
            E-mail
          </label>
        </div>
      </div>
      <button
        aria-autocomplete="none"
        aria-controls="radix-:r6:"
        aria-expanded="false"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 bg-white-lilac-50 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 p-3 dark:bg-smoke-950"
        data-state="closed"
        dir="ltr"
        role="combobox"
        type="button"
      >
        <span
          style="pointer-events: none;"
        >
          N√£o
        </span>
        <span
          aria-hidden="true"
        >
          <svg
            aria-hidden="true"
            class="ms-2.5 size-2.5"
            fill="none"
            viewBox="0 0 10 6"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="m1 1 4 4 4-4"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </span>
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
        >
          Estrangeiro
        </label>
      </button>
      <select
        aria-hidden="true"
        style="position: absolute; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; word-wrap: normal;"
        tabindex="-1"
      >
        <option
          value="Sim"
        >
          Sim
        </option>
        <option
          selected=""
          value="N√£o"
        >
          N√£o
        </option>
      </select>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Passport"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":r7:"
            required=""
            type="number"
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":r7:"
          >
            Passaporte
          </label>
        </div>
      </div>
      <button
        aria-controls="radix-:r8:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="group text-left text-gray-800 outline-none"
        data-state="closed"
        type="button"
      >
        <div
          class="relative flex h-min flex-col"
        >
          <div
            class="relative"
          >
            <input
              autocomplete="on"
              class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 py-3 pb-3 pl-3 pr-8 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100 cursor-pointer group-focus:border-primary-300 group-data-[state='open']:border-primary-300"
              id=":r9:"
              readonly=""
              value="29/02/2024"
            />
            <label
              class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
              for=":r9:"
            >
              Data de Nascimento
            </label>
            <div
              class="absolute right-2 top-1/2 -translate-y-1/2 p-1 text-comet-500 transition-all dark:text-dark-300"
            >
              <svg
                fill="none"
                height="18"
                viewBox="0 0 24 24"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 2V5"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M16 2V5"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M3.5 9.09H20.5"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M21 8.5V17C21 20 19.5 22 16 22H8C4.5 22 3 20 3 17V8.5C3 5.5 4.5 3.5 8 3.5H16C19.5 3.5 21 5.5 21 8.5Z"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-miterlimit="10"
                  stroke-width="1.5"
                />
                <path
                  d="M11.9955 13.7H12.0045"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
                <path
                  d="M8.29431 13.7H8.30329"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
                <path
                  d="M8.29431 16.7H8.30329"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </div>
          </div>
        </div>
      </button>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Cell Phone"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":ra:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":ra:"
          >
            Celular
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Phone"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":rb:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":rb:"
          >
            Telefone
          </label>
        </div>
      </div>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Contact Name"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":rc:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":rc:"
          >
            Nome de Contato
          </label>
        </div>
      </div>
      <button
        aria-autocomplete="none"
        aria-controls="radix-:rd:"
        aria-expanded="false"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 bg-white-lilac-50 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 p-3 dark:bg-smoke-950"
        data-state="closed"
        dir="ltr"
        role="combobox"
        type="button"
      >
        <span
          style="pointer-events: none;"
        >
          N√£o
        </span>
        <span
          aria-hidden="true"
        >
          <svg
            aria-hidden="true"
            class="ms-2.5 size-2.5"
            fill="none"
            viewBox="0 0 10 6"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="m1 1 4 4 4-4"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
            />
          </svg>
        </span>
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
        >
          Tem Defici√™ncia?
        </label>
      </button>
      <select
        aria-hidden="true"
        style="position: absolute; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; word-wrap: normal;"
        tabindex="-1"
      >
        <option
          value="Sim"
        >
          Sim
        </option>
        <option
          selected=""
          value="N√£o"
        >
          N√£o
        </option>
      </select>
      <div
        class="relative flex h-min flex-col w-full"
      >
        <div
          class="relative"
        >
          <input
            aria-label="Social Name"
            autocomplete="on"
            class="peer w-full rounded-lg border-2 bg-transparent font-normal text-comet-500 outline-none transition-all placeholder:opacity-0 dark:border-shark-950 dark:text-white-lilac-50 p-3 focus:border-primary-300 dark:focus:border-primary-400 focus:placeholder:opacity-100"
            id=":re:"
            required=""
            value=""
          />
          <label
            class="pointer-events-none absolute left-4 top-1/2 z-10 -translate-y-1/2 px-[3px] text-sm text-comet-500 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all peer-valid:top-[-35%] peer-valid:translate-y-1/2 peer-read-only:top-[-35%] peer-read-only:translate-y-1/2 peer-focus:top-[-35%] peer-focus:translate-y-1/2 dark:text-dark-300 before:dark:bg-smoke-950 peer-focus:text-primary-400"
            for=":re:"
          >
            Nome Social
          </label>
        </div>
      </div>
      <button
        aria-controls="radix-:rf:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 p-3 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 bg-transparent dark:bg-transparent"
        data-state="closed"
        type="button"
      >
        Selecione o Pa√≠s
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950"
        >
          Pa√≠s
        </label>
      </button>
      <button
        aria-controls="radix-:rg:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 p-3 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 bg-transparent dark:bg-transparent"
        data-state="closed"
        type="button"
      >
        Selecione a Cidade
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950"
        >
          Cidade Natal
        </label>
      </button>
      <button
        aria-controls="radix-:rh:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 p-3 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 bg-transparent dark:bg-transparent"
        data-state="closed"
        type="button"
      >
        Selecione o G√™nero
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950"
        >
          G√™nero
        </label>
      </button>
      <button
        aria-controls="radix-:ri:"
        aria-expanded="false"
        aria-haspopup="dialog"
        class="group relative flex items-center justify-between rounded-lg border-2 border-zumthor-100 p-3 text-sm font-medium text-comet-500 transition-all hover:bg-porcelain-50 focus:outline-none dark:border-shark-950 dark:text-dark-300 dark:hover:opacity-60 focus:border-primary-300 dark:focus:border-primary-400 dark:data-[state="open"]:border-primary-400 data-[state="open"]:border-primary-300 focus:placeholder:opacity-100 bg-transparent dark:bg-transparent"
        data-state="closed"
        type="button"
      >
        Selecione o Estado Civil
        <label
          class="pointer-events-none absolute left-4 top-[-47%] z-10 translate-y-1/2 px-[3px] text-sm text-comet-400 transition-all before:absolute before:inset-0 before:-z-10 before:block before:w-full before:bg-white-lilac-50 before:transition-all group-focus:text-primary-300 group-data-[state="open"]:text-primary-400 dark:text-dark-300 before:dark:bg-smoke-950"
        >
          Estado Civil
        </label>
      </button>
    </div>
    <div
      class="flex w-full justify-center sm:justify-end"
    >
      <button
        aria-label="Update User"
        class="relative transition-transform duration-200 ease-in-out hover:scale-105 focus:scale-105 active:scale-90 min-w-[12.404rem]"
        type="button"
      >
        <div
          aria-label="Button Background"
          class="absolute size-full rounded-md opacity-[16%] bg-success-500"
        />
        <div
          class="flex h-full items-center justify-center gap-2 px-5 py-[0.625rem] flex-row text-success-500"
        >
          <span
            class="z-10 font-medium leading-[1.125rem] tracking-[0.027rem] text-[0.9375rem]"
          >
            Criar Colaborador
          </span>
        </div>
      </button>
    </div>
  </form>
</div>
`;
